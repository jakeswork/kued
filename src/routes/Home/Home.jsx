import React from 'react';
import { FiArrowRight } from 'react-icons/fi';

import propTypes from './types';
import Text from '../../components/Text';
import Button from '../../components/Button';
import TopNavigation from './components/TopNavigation';

const Home = ({ classes }) => (
  <header className={classes.root}>
    <TopNavigation />
    <main className={classes.center}>
      <Text h1 style={{ color: 'white' }}>
        Elevate your arena experience.
      </Text>
      <Text h4 style={{ color: '#b1a0a0', margin: 16, textAlign: 'center' }}>
        With the most up-to-date live meta data, we allow you to
        <br />
        predict opponents and open up the best opportunities.
      </Text>
      <div className={classes.buttonsContainer}>
        <Button link="/feed" icon={<FiArrowRight />}>Get Started</Button>
        <Button secondary style={{ color: 'white' }} link="/contact">Learn More</Button>
      </div>
    </main>
  </header>
);

Home.propTypes = propTypes;

export default Home;
