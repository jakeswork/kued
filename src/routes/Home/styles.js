import backgroundImage from '../../assets/background-image.jpg';

export default theme => ({
  root: {
    display: 'flex',
    height: '100vh',
    justifyContent: 'center',
    alignItems: 'center',
    background: theme.colorSecondary,
    backgroundImage: `url(${backgroundImage})`,
    backgroundSize: 'cover',
    backgroundRepeat: 'no-repeat',
    backgroundPosition: 'top',
  },
  center: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    textAlign: 'center',
  },
  buttonsContainer: {
    width: 300,
    padding: '2em',
    display: 'flex',
    justifyContent: 'space-between',
  },
});
