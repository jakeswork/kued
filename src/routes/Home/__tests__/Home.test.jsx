import React from 'react';
import { shallow } from 'enzyme';

import Home from '../Home';

const defaultProps = {};

describe('The Home instance', () => {
  describe('when called with default props', () => {
    it('should render without throwing errors', () => {
      const wrapper = shallow(<Home {...defaultProps} />);

      expect(wrapper).toMatchSnapshot();
    });
  });
});
