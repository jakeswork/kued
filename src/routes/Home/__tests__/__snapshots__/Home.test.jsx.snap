// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Home instance when called with default props should render without throwing errors 1`] = `
<div>
  Hello, World!
</div>
`;
